Dark theme

viviicat opened the pull request:
Resolves #1972

mouse-reeve commented:
I changed the base of this PR to merge into #1975 -- the main change is that I renamed your files. I hope I didn't totally jank up your process! If I did we can create a new PR without my changes, but hopefully this works and gets our branches happily in sync

viviicat commented:
> I changed the base of this PR to merge into #1975 -- the main change is that I renamed your files. I hope I didn't totally jank up your process! If I did we can create a new PR without my changes, but hopefully this works and gets our branches happily in sync

For some reason I am not able to set up the dev environment anymore on this branch. I get 

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "bookwyrm_theme" does not exist
LINE 1: SELECT "bookwyrm_theme"."path" FROM "bookwyrm_theme"
                                            ^
```

even if I try to run `./bw-dev migrate`

mouse-reeve commented:
I think I fixed that in the branch latest, try it now?
